docs/helm-chart/production-guide.rst

```diff
@@ -205,8 +205,9 @@ You can change the Service type for the webserver to be ``LoadBalancer``, and se
 .. code-block:: yaml
 
   webserver:
-    service: LoadBalancer
-    annotations: {}
+    service: 
+      type: LoadBalancer
+      annotations: {}
```

Comment:
```suggestion
```

We can probably remove the empty annotations while we are at it, that's the default.